main.go

```diff
@@ -112,14 +115,23 @@ func initServer() *http.Server {
 
 	config := mustLoadConfig()
 	packagesBasePaths := getPackagesBasePaths(config)
-	ensurePackagesAvailable(ctx, packagesBasePaths)
+
+	indexers := []Indexer{
+		util.NewFileSystemIndexer(packagesBasePaths...),
+	}
+	if serveZip {
```

Comment:
I think this will be safe, I only added it last minute as a safety net to don't have it enabled by default in production by now :slightly_smiling_face: Not a strong opinion, so as you prefer :slightly_smiling_face: